backend/webhook/publisher/src/main/java/co/airy/core/webhook/publisher/Mapper.java

```diff
@@ -19,16 +21,23 @@
     }
 
     public WebhookBody fromMessage(Message message) throws Exception {
-        contentMapper.render(message);
+        final Content content = contentMapper.renderWithDefaultAndLog(message);
+
+        if (!(content instanceof Text)) {
+            throw new NotATextMessage();
+        }
 
         return WebhookBody.builder()
                 .conversationId(message.getConversationId())
                 .id(message.getId())
-                .text(message.getContent())
+                .text(((Text) content).getText())
                 .source(message.getSource())
                 .postback(buildPostback(message))
                 .sentAt(isoFromMillis(message.getSentAt()))
-                .sender(new WebhookBody.Sender(message.getSenderId()))
+                .sender(WebhookBody.Sender.builder()
+                        .id(message.getSenderId())
+                        .type(message.getSenderType().toString().toLowerCase())
```

Comment:
Sources like Rasa need the sender type to know if they need to react to this message.